### PR TITLE
refactor(TransactionSumAddress): code

### LINKED ISSUE #412: transaction-sum-address

### PR DESCRIPTION
**Pull request by samandarkuchkarov**
Fixes #412 

## Proposed Changes

  -Change the imports to absolute:
  -Change the colors from the constants to similar ones in Color
  -Replace svg icons with icon icons.

## Checklist

- [x] I have read the rules of [**contribution**](https://github.com/haqq-network/haqq-wallet/blob/main/docs/contribution.md#important-steps).

### CODE REVIEW

File: src/components/transaction-address.tsx [hunk 1]
```diff
@@ -122,24 +120,24 @@ export const TransactionAddress = ({
           address === '' ? (
             <View style={page.inputButtonContainer}>
               <IconButton onPress={onPressPaste}>
-                <PasteIcon
-                  color={LIGHT_GRAPHIC_GREEN_1}
-                  width={25}
-                  height={25}
+                <Icon
+                  style={page.icon}
```
Comment (gHashTag) [inline src/components/transaction-address.tsx:124]: Use i24(by design) and remove `icon` style

File: src/components/transaction-address.tsx [hunk 2]
```diff
@@ -122,24 +120,24 @@ export const TransactionAddress = ({
...
                 />
               </IconButton>
               <Spacer width={12} />
               <IconButton onPress={onPressQR}>
-                <QRScanner
-                  color={LIGHT_GRAPHIC_GREEN_1}
-                  width={25}
-                  height={25}
+                <Icon
+                  style={page.icon}
```
Comment (gHashTag) [inline src/components/transaction-address.tsx:132]: And rename `page` to `style`

**Review by gHashTag — CHANGES_REQUESTED** [review]
Fix comments

File: src/components/list-contact.tsx [hunk 3]
```diff
@@ -28,23 +23,24 @@ export const ListContact = ({
   ...flatListProps
 }: ListContactProps) => {
   const {bottom} = useSafeAreaInsets();
+  const listFooterComponentRender = () => <Spacer height={bottom} />;
   return (
     <FlatList
       data={data}
-      ListFooterComponent={() => <Spacer height={bottom} />}
+      ListFooterComponent={listFooterComponentRender}
```
Comment (vivalaakam) [inline src/components/list-contact.tsx:30]: that difference with previous version?

**Review by vivalaakam — APPROVED** [review]

**Pull request merged**